packages/databyss-editor/components/Element.js

```diff
@@ -43,7 +54,11 @@ const Element = ({ attributes, children, element }) => {
           position="relative"
           justifyContent="center"
         >
-          {block.__showNewBlockMenu && (
+          {/* on initialization block does not have '__showNewBlockMenu' */
+          (block.__showNewBlockMenu ||
```

Comment:
This seems weird. We should initialize the `__showNewBlockMenu` appropriately when we are loading/creating the page